### ISSUE TITLE
How to make long notes or ties

### ISSUE THREAD

**Issue by pabloriera** [cuthbertLab/music21#216]
Hi,

I notice that music21.duration.Duration(x) with x greater than 14 doesn't work when exporting a midi file.

```
TranslateException                        Traceback (most recent call last)
<ipython-input-223-9ab634ec8085> in <module>()
     17 sc = music21.stream.Score(s)
     18 midi_file = open('test.mid','wb')
---> 19 midi_file.write(music21.midi.translate.streamToMidiFile(sc).writestr())
     20 midi_file.close()
     21 

/usr/local/lib/python2.7/dist-packages/music21/midi/translate.pyc in streamToMidiFile(inputM21)
   1901 
   1902     s = inputM21
-> 1903     midiTracks = streamHierarchyToMidiTracks(s)
   1904 
   1905     # update track indices

/usr/local/lib/python2.7/dist-packages/music21/midi/translate.pyc in streamHierarchyToMidiTracks(inputM21, acceptableChannelList)
   1796         channelForInstrument=channelForInstrument,
   1797         channelsDynamic=channelsDynamic,
-> 1798         initChannelForTrack=initChannelForTrack)
   1799 
   1800     #environLocal.printDebug(['got netPackets:', len(netPackets), 'packetStorage keys (tracks)', packetStorage.keys()])

/usr/local/lib/python2.7/dist-packages/music21/midi/translate.pyc in _processPackets(packets, channelForInstrument, channelsDynamic, initChannelForTrack)
   1239                     break
   1240             if ch is None:
-> 1241                 raise TranslateException('no unused channels available for microtone/instrument assignment')
   1242             p['midiEvent'].channel = ch
   1243             # change channel of note off; this is used above to turn off pbend

TranslateException: no unused channels available for microtone/instrument assignment
```

How can I create longer notes? 

Thanks

**Comment by mscuthbert** [comment]
Works fine for me when using the main MIDI write routines:

```
In [20]: s = stream.Stream()

In [21]: n = note.Note(quarterLength=20)

In [22]: n.show('midi')

In [23]: n.write('midi')
Out[23]: '/var/folders/1t/0_91h1r94kg21qt1jt20lxkc0000gn/T/music21/tmpf6qohv4s.mid'

In [24]: n.duration
Out[24]: <music21.duration.Duration 20.0>

In [25]: s.append(n)

In [26]: s.show('midi')

In [27]: s.write('midi')
Out[27]: '/var/folders/1t/0_91h1r94kg21qt1jt20lxkc0000gn/T/music21/tmpaf0k96ld.mid'
```

**Comment by mscuthbert** [comment]
Fixed with fix for #223
